src/components/Researcher/SaveResearcherData.tsx

```diff
@@ -17,7 +17,7 @@ export const fetchResult = async (authString, id, type, modal) => {
 }
 
 export const fetchPostData = async (authString, id, type, modal, methodType, bodyData) => {
-  const baseUrl = "https://" + LAMP.Auth._auth.serverAddress
+  const baseUrl = "https://" + (!!LAMP.Auth._auth.serverAddress ? LAMP.Auth._auth.serverAddress : "api.lamp.digital")
```

Comment:
In the future, please convert this outdated syntax to the newer TypeScript [nullish coalescing operator.](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing)

Reply:
@avaidyam I was trying that first, but the server address was actually returning a blank string, it was not passing this '??' operator checking. That is why used this one. We are using the nullish coalescing operator wherever possible.